### PR TITLE
Feature - Sortable table components

### PR DESCRIPTION
**Pull request by ShahStavan**

### CODE REVIEW

**Comment by ShahStavan** [comment]
Hii @markmead 
I was facing some issues with that version of Next that's why I changed the version. 
I would appreciate it if you could review the most recent PR, as the default Next version has been retained. Please provide feedback on this decision. 
Thank you.

File: package-lock.json
Comment (markmead) [inline package-lock.json:1]: Project uses `yarn` not `npm` commands. Please remove this and use `yarn`.

File: public/components/application-ui-tables/8-dark.html
Comment (markmead) [inline public/components/application-ui-tables/8-dark.html:1]: Let's not worry about the dark mode component at the moment.

File: public/components/application-ui-tables/8.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <td class="whitespace-nowrap px-4 py-2 font-medium text-gray-900">Purvesh Shah</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">24/05/1995</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">Singer</td>
+        <td class="whitespace-nowrap px-4 py-2 text-gray-700">$150,000</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<script>
```
Comment (markmead) [inline public/components/application-ui-tables/8.html:51]: Shouldn't be any JavaScript in base components. There are 3 variants:

- Base (HTML)
- Dark (HTML)
- Interactive (HTML + Alpine JS)

Let's get the base component where it needs to be before tackling anything else.

File: yarn.lock
Comment (markmead) [inline yarn.lock:1]: Weird to have a change in this file...

**Review by markmead — CHANGES_REQUESTED** [review]
Not having any issues with the current setup so no idea why you are, odd! Few things to address. 👍

**Pull request closed without merging**